src/App.js

```diff
@@ -9,14 +8,40 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      movie: null
+      movies: [],
+      // movies: movieData.movies,
+      movie: null,
+      error: ''
     }
   }
 
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => {
+        // console.log(response)
+        if(!response.ok) {
+          throw new Error("Oopsies! Something went wrong 🤡")
```

Comment:
nice job throwing a new Error here! ❗️

Reply:
+1, really great work adding those checks for error handling!  This is perfect so that now your `.catch()` is handling any status codes outside of the 200s.  The only other small refactor you could do here is potentially moving this fetch request inside of an `apiCalls.js` file and then importing it/calling it here!